basic/src/test/java/org/jboss/hal/testsuite/test/configuration/requestcontroller/RequestControllerSubsystemTestCase.java

```diff
@@ -0,0 +1,111 @@
+package org.jboss.hal.testsuite.test.configuration.requestcontroller;
+
+import org.jboss.arquillian.drone.api.annotation.Drone;
+import org.jboss.arquillian.graphene.page.Page;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.hal.testsuite.category.Shared;
+import org.jboss.hal.testsuite.creaper.ManagementClientProvider;
+import org.jboss.hal.testsuite.page.config.RequestControllerPage;
+import org.jboss.hal.testsuite.util.ConfigChecker;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.WebDriver;
+import org.wildfly.extras.creaper.core.online.ModelNodeResult;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.ReadAttributeOption;
+import org.wildfly.extras.creaper.core.online.operations.admin.Administration;
+
+import java.io.IOException;
+import java.util.concurrent.TimeoutException;
+
+@RunWith(Arquillian.class)
+@Category(Shared.class)
+public class RequestControllerSubsystemTestCase {
+
+    @Drone
+    private WebDriver browser;
+
+    @Page
+    private RequestControllerPage page;
+
+    private static final Address REQUEST_CONTROLLER_SUBSYSTEM_ADDRESS = Address.subsystem("request-controller");
+
+    private static final String
+            ACTIVE_REQUESTS = "active-requests",
+            MAX_REQUESTS = "max-requests",
+            TRACK_INDIVIDUAL_ENDPOINTS = "track-individual-endpoints";
+
+    private static final OnlineManagementClient client = ManagementClientProvider.createOnlineManagementClient();
+    private static final Administration administration = new Administration(client);
+    private static final Operations operations = new Operations(client);
+
+    @Before
+    public void before() {
+        page.navigate();
+    }
+
+    @AfterClass
+    public static void afterClass() throws InterruptedException, TimeoutException, IOException {
+        administration.reloadIfRequired();
+    }
+
+    @Test
+    public void verifyActiveRequests() throws Exception {
```

Comment:
This name looks more as some internal method name than test name, I suggest renaming it to something like `activeRequestsTest` or something similar.